app/scripts/modules/core/src/navigation/state.provider.ts

```diff
@@ -31,12 +31,8 @@ export class StateConfigProvider implements IServiceProvider {
   private root: INestedState = {
     name: 'home',
     abstract: true,
-    url: '?{debug:boolean}&{vis:boolean}&{trace:query}',
-    params: {
-      debug: { dynamic: true },
-      vis: { dynamic: true, value: false, squash: true },
-      trace: { dynamic: true, value: null, squash: true },
-    },
+    url: '?{debug:boolean}&{vis:query}&{trace:query}',
+    dynamic: true,
```

Comment:
latest ui-router supports `dynamic` on the state def instead of individually on each parameter.

@erikmunson @archana-s @jrsquared @danielpeach @sbwsg FYI